<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealWindow</class>
 <widget class="QDialog" name="DealWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidgetDeals">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Active">
      <attribute name="title">
       <string>Active</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QTableView" name="tableViewDealsActive">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="searchLE">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxDeals1">
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string notr="true" comment="Select field to sort:" extracomment="Select field to sort:">Select field to sort:</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>deal_id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>itin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rentStartDate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rentEndDate</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Closed">
      <attribute name="title">
       <string>Closed</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QTableView" name="tableViewDealsClosed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="searchLE2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Search:</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxDeals2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="placeholderText">
          <string comment="Select field to sort:" extracomment="Select field to sort:">Select field to sort:</string>
         </property>
         <item>
          <property name="text">
           <string>deal_id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>itin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rentStartDate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>rentEndDate</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addDealButton">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/icons/Resources/plus.png</normaloff>:/new/icons/Resources/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="paymentButton">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/new/icons/Resources/free-icon-font-usd-square-7653268.png</normaloff>:/new/icons/Resources/free-icon-font-usd-square-7653268.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
